.gitlab-ci.yml

```diff
@@ -362,6 +362,26 @@ ci-finmap-dev:
   variables:
     COQ_VERSION: "dev"
 
+# The FCSL-PCM library
```

Comment:
I will soon be preparing the release for math-comp 1.10.0.  I don't expect this Pull Request to be part of this release.  Please react if my expectation is wrong.

Reply:
I think this is fine. It would be great though to merge this PR before merging some other PRs that may introduce breaking changes, e.g. #429.